duckduckgo/duckduckgo-privacy-extension
# Remove unused element hiding code

**Reviewer:** @Charlie-belmer 

**CC:** @kdzwinel @jdorweiler 


## Description:
Remove outdated element hiding code.


## Steps to test this PR:
1. Build and install extension in Chrome
2. Visit a website with extension installed
3. Open dev tools, then go to the Sources tab
4. In the upper left of the Sources tab select Content scripts
5. Verify that there is no element-hiding.js script injected by extension


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
